src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -124,6 +123,9 @@ export function useRefetchQuoteCallback() {
     async (params: RefetchQuoteCallbackParmams) => {
       const { sellToken, buyToken, amount, chainId } = params.quoteParams
       try {
+        // we need to signal to other parts of the app
+        // that we are loading a new quote
+        loadingQuote(true)
```

Comment:
I feel like a better name for this would be: `setIsQuoteLoading` or at a bare minimum `setLoadingQuote`

Reply:
meeehhhhhhhh